src/Akeneo/CouplingDetector/Configuration/examples/akeneo_pim-community-dev.php_cd

```diff
@@ -96,172 +98,243 @@ $rules = [
     ),
 ];
 
+// LEGACY can be fixed by moving a class
+// DEPRECATED can be fixed by removing the class when we start to work on last supported version
+$exclusions = [
```

Comment:
shouldn't they be defined in the rules themselves? 
I don't understand the pros of defining them alone